R2API/Utils/Reflection.cs

```diff
@@ -923,5 +925,81 @@ public static class Reflection {
         }
 
         #endregion
+
+        public static System.Reflection.FieldInfo GetNestedField(Type type, string fieldName) {
+            var nestedTypes = type.GetNestedTypes((System.Reflection.BindingFlags)(-1));
```

Comment:
Please use proper BindingFlags

Reply:
When I was trying to figure out how to get a nested field info, iDeath linked me to a previous conversation in the server talking about the same issue. Reading through that conversation, this was their solution. I didn't even question it. Would it be better to pass the list of all binding flags instead?